extensions/ql-vscode/src/sarif-parser.ts

```diff
@@ -1,33 +1,34 @@
 import * as Sarif from 'sarif';
 import * as fs from 'fs-extra';
-import { parser } from 'stream-json';
-import { pick } from 'stream-json/filters/Pick';
 import Assembler = require('stream-json/Assembler');
-import { chain } from 'stream-chain';
 import { getErrorMessage } from './pure/helpers-pure';
+import Pick = require('stream-json/filters/Pick');
```

Comment:
Is there a reason why you are using `require` instead of standard imports?

Reply:
Hm, I had thought that I needed to import it this way as a namespace to use the `Pick.withParser()` method (similar to the way `Assembler` was imported to use `Assembler.connectTo`

Reply:
I think this will work:

```suggestion
import * as pick from 'stream-json/filters/Pick';
```

Or possibly better:

```suggestion
import { withParser } from 'stream-json/filters/Pick';
```

Reply:
Ah yes.. i'll do the same for `connectTo` from `Assembler` as well.